integration/networking/nat_windows_test.go

```diff
@@ -0,0 +1,114 @@
+package networking
+
+import (
+	"context"
+	"net"
+	"testing"
+	"time"
+
+	containertypes "github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/integration/internal/container"
+	"github.com/docker/docker/integration/internal/network"
+	"github.com/docker/docker/testutil"
+	"github.com/docker/go-connections/nat"
+	"gotest.tools/v3/assert"
+	is "gotest.tools/v3/assert/cmp"
+)
+
+// TestBridgeICCWindows tries to ping container ctr1 from container ctr2 using its hostname.
+// Checks DNS resolution, and whether containers can communicate with each other.
+// Regression test for https://github.com/moby/moby/issues/47370
+func TestBridgeICCWindows(t *testing.T) {
```

Comment:
Now that we split, should this test also be named something "nat"?

Reply:
Good point! I should have called it `nat`-something when I first wrote it. Renamed now.